api/v1/certificatepolicy_types.go

```diff
@@ -69,73 +66,120 @@ func (t Target) String() string {
 	return fmt.Sprintf(fmtSelectorStr, t.Include, t.Exclude, *t.MatchLabels, *t.MatchExpressions)
 }
 
-// CertificatePolicySpec defines the desired state of CertificatePolicy
+// CertificatePolicySpec defines the certificates to be checked and the parameters within which
+// those certificates should fall.
 type CertificatePolicySpec struct {
-	// Only Inform is currently supported. Setting this to Enforce will have the same effect as Inform.
-	// +kubebuilder:validation:Enum=Inform;inform;Enforce;enforce
+	// Set to "Inform". Enforce is not currently supported, so setting this to Enforce will have the
+	// same effect as Inform.
 	RemediationAction RemediationAction `json:"remediationAction,omitempty"`
-	// 'namespaceSelector' defines the list of namespaces to include/exclude for objects defined in
-	// spec.objectTemplates. All selector rules are ANDed. If 'include' is not provided but
-	// 'matchLabels' and/or 'matchExpressions' are, 'include' will behave as if ['*'] were given. If
-	// 'matchExpressions' and 'matchLabels' are both not provided, 'include' must be provided to
+
+	// NamespaceSelector defines the list of namespaces to include/exclude for objects defined in
+	// spec.objectTemplates. All selector rules are ANDed. If Include is not provided but
```

Comment:
What is ANDed

Reply:
As in all results are combined using a logical "AND" operation.